### PR TITLE
feat(docs):  batch 8 of proofread/edited docs

### PR DESCRIPTION
**Pull request by vwheeler63**
cc:  @kisvegabor 

- tabview.rst  [link](http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/tabview.html)
- textarea.rst  [link](http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/textarea.html)
- tileview.rst  [link](http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/tileview.html)
- win.rst  [link](http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/win.html)

Current (generated) versions of these documents can be found at the links provided above.

Most of these edits are to correct grammar and clarify meaning. Some homework had to be done by digging in the C code to clarify or provide information I felt should be there to make LVGL programmers' lives easier.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.  n/a
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.  n/a
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.  n/a
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).  n/a
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).  n/a
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.  Done.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.  Done.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).  Unable.

### CODE REVIEW

**Review by kisvegabor — DISMISSED** [review]
I could just quickly scan it and it looked good. @liamHowatt please take a deeper look.

**Comment by vwheeler63** [comment]
@kisvegabor @liamHowatt Hi, Gentlemen!  In case it helps, this PR is my highest priority since other things are waiting on it.

File: docs/details/widgets/tabview.rst [hunk 1]
```diff
@@ -1,85 +1,93 @@
...
-Set tab bar size
-----------------
+Setting tab bar size
+--------------------
 
 The size of the tab bar can be adjusted by :cpp:expr:`lv_tabview_set_tab_bar_size(tabview, size)`
-In case of vertical arrangement is means the height of the tab bar, and in horizontal
-arrangement it means the width.
+When tabs are on the top or bottom, this means the height of the tab bar, and when
+they are on the sides, it means the width.
```
Comment (liamHowatt) [inline docs/details/widgets/tabview.rst:82]: This adds some much needed clarity.

File: docs/details/widgets/textarea.rst [hunk 2]
```diff
@@ -160,14 +218,17 @@ Events
...
--  :cpp:enumerator:`LV_EVENT_VALUE_CHANGED` Sent when the content of the text area has
-   been changed.
--  :cpp:enumerator:`LV_EVENT_READY` Sent when :cpp:enumerator:`LV_KEY_ENTER` is pressed (or sent) to
-   a one line text area.
+   :cpp:expr:`lv_textarea_set_insert_replace(textarea, "New text")` can be called
+   from within that event to replace the text to be inserted.  The contents of the
+   buffer passed must be survive long enough for the call to `lv_timer_handler()`
+   that is driving the event to return (after which the Text Area's label will have
+   copied the text).  So it should not be a local buffer (created on the stack)
+   where its contents an be destroyed before that happens.  Passing ``""`` means "do
```
Comment (liamHowatt) [inline docs/details/widgets/textarea.rst]:
```suggestion
   where its contents will be destroyed before that happens.  Passing ``""`` means "do
```

File: docs/details/widgets/win.rst [hunk 3]
```diff
@@ -4,58 +4,89 @@
...
 Usage
 *****
 
+
 Create a Window
 ---------------
 
-:cpp:expr:`lv_win_create(parent, header_height)` creates a Window with an empty
-header.
+:cpp:expr:`lv_win_create(parent)` creates a Window that, is initially
```
Comment (liamHowatt) [inline docs/details/widgets/win.rst]:
```suggestion
:cpp:expr:`lv_win_create(parent)` creates a Window that is initially
```

File: docs/details/widgets/tabview.rst (hunk 1 above)
Comment (vwheeler63) [reply docs/details/widgets/tabview.rst:82]: > This adds some much needed clarity.

Agreed!

**Comment by vwheeler63** [comment]
@liamHowatt Nice catches!  Changes accepted.

cc:  @kisvegabor

**Review by kisvegabor — APPROVED** [review]
:+1:

**Review by liamHowatt — APPROVED** [review]

**Comment by vwheeler63** [comment]
@liamHowatt Thank you!

**Pull request merged**